internal/service/alertconfiguration/data_source_alert_configuration_test.go

```diff
@@ -41,23 +36,19 @@ func TestAccConfigDSAlertConfiguration_basic(t *testing.T) {
 
 func TestAccConfigDSAlertConfiguration_withThreshold(t *testing.T) {
 	var (
-		alert          = &admin.GroupAlertsConfig{}
-		dataSourceName = "data.mongodbatlas_alert_configuration.test"
-		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		projectName    = acc.RandomProjectName()
+		projectID = acc.ProjectIDExecution(t)
 	)
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
```

Comment:
It'd be great to show in the PR description the impact of running these tests sequentially vs in parallel: this info should be easy to get, just check the time needed to run the tests in this PR vs in the last nightly run

Reply:
as I commented to Marco the "config" test group that runs database_user and alert_notification is taking less time now than before even being in serial.  anyway i'm going to follow Agustin idea of just running in serial the plural ds tests.